apps/checkout/src/sections/CheckoutForm.tsx

```diff
@@ -24,14 +29,41 @@ export const CheckoutForm = () => {
   const methods = useForm<FormData>({ resolver, mode: "onBlur" });
   const { setValue, watch } = methods;
 
+  const [selectedValue, setSelectedValue] = useState<string>();
+
   const handleEmailChange = (value: string) => setValue("email", value);
 
+  const addressData: Partial<AddressFragment> = {
```

Comment:
you might want to use some fake data for that

Reply:
It's just for preview purposes for this pr, we can just remove it altogether in the next pr